easybuild/easyconfigs/m/MultiQC/MultiQC-1.7-foss-2018b-Python-3.6.6.eb

```diff
@@ -95,7 +95,12 @@ exts_list = [
     }),
     ('multiqc', version, {
```

Comment:
@SethosII Patch is still expected to use `multiqc` because the extension name used here...

If you change the extension name to `MultiQC`, you'll also need to change the name of the source tarball (using `source_tmpl': `%(namelower)s-%(version)s.tar.gz'`).

The alternative is to rename the patch file to use `multiqc`